tla/README.md

```diff
@@ -5,3 +5,9 @@ This directory contains some formal specifications of CCF. For more information,
 You can also interact with these specifications using codespaces:
 
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=180112558&machine=xLargePremiumLinux&devcontainer_path=.devcontainer%2Ftlaplus%2Fdevcontainer.json&location=WestEurope)
+
+## Trace validation
+
+You can produce fresh traces quickly from the driver by running the `make_traces.sh` script from this directory.
+
+Calling the trace validation on, for example, the `startup` scenario can then be done with `JSON=../build/startup.ndjson ./tlc.sh consensus/Traceccfraft.tla`.
```

Comment:
I tested this out (fresh codespace from this PR) and `startup.ndjson` does not exist
![image](https://github.com/microsoft/CCF/assets/1835251/35a58c3b-9037-4b83-8a13-f3b1edbb849d)